extensions/binding/org.eclipse.smarthome.binding.wemo/src/main/java/org/eclipse/smarthome/binding/wemo/discovery/WemoDiscoveryService.java

```diff
@@ -85,16 +87,17 @@
 			public void run() {
 				discoverWemo();		}
 	};
-			
+
 	public WemoDiscoveryService() {
 		super(WemoHandler.SUPPORTED_THING_TYPES, 15, true);
 	}
- 
+
     public Set<ThingTypeUID> getSupportedThingTypeUIDs() {
         return WemoHandler.SUPPORTED_THING_TYPES;
     }
- 
-	protected void startScan() {
+
+	@Override
```

Comment:
Oh, weird. it looks like my Eclipse Installer workspace didn't apply the ESH formatting. 
Let me try fixing the formatting issues + read the PR conventions.